lib/ansible/cli/galaxy.py

```diff
@@ -305,9 +305,9 @@ def execute_info(self):
                 role_info.update(role_spec)
 
             data = self._display_role_info(role_info)
-            ### FIXME: This is broken in both 1.9 and 2.0 as
-            # _display_role_info() always returns something
-            if not data:
+            
+            role_location = os.path.join(roles_path[0], role)
```

Comment:
Does roles_path ever store more than one string? Hadn't noticed that with any of the roles I've installed and my interaction with the code.